Standard/ECDb/ECDbMap.ecschema.md

```diff
@@ -0,0 +1,183 @@
+---
+noEditThisPage: true
+Schema: ECDbMap
+Warning: This file was automatically generated via ecjson2md. Do not edit this file. Any edits made to this file will be overwritten the next time it is generated.
+---
+
+# ECDbMap (ECDb DB Mapping) [!badge text="Schema" kind="Info"]
+
+**Alias:** ecdbmap
+
+**Version:** 2.0.2
+
+Custom attributes that customize ECDb's ECSchema to database mapping.
+
+## Table of contents
+- [Struct Classes](#struct-classes)
+	- [DbIndex](#dbindex)
+- [Custom Attribute Classes](#custom-attribute-classes)
+	- [ClassMap](#classmap)
+	- [DbIndexList](#dbindexlist)
+	- [ForeignKeyConstraint](#foreignkeyconstraint)
+	- [ImportRequiresVersion](#importrequiresversion)
+	- [JoinedTablePerDirectSubclass](#joinedtableperdirectsubclass)
+	- [LinkTableRelationshipMap](#linktablerelationshipmap)
+	- [PropertyMap](#propertymap)
+	- [SchemaMap](#schemamap)
+	- [ShareColumns](#sharecolumns)
+	- [UseRequiresVersion](#userequiresversion)
+
+### **DbIndex** *Sealed* [!badge text="StructClass" kind="info"] [<img src=".././media/imodel-schema-editor-icon.png">](https://imodelschemaeditor.bentley.com/?stage=browse&elementtype=StructClass&id=ECDbMap.DbIndex)
```

Comment:
I don't think you need to use the `ecjson2md` tool at this point. We use it at part of our build pipeline to automatically convert and generate the relevant docs.

Reply:
Earlier, I thought you want to run it locally to test the documentation..

Reply:
Ah okay I understand. I don't think the documentation mentions this, I just saw every other schema has these files so I thought I had to put them there. It should not hurt to leave them in this PR I guess. But thanks for explaining!

Reply:
Yes, I think at the moment we don't have a concrete policy for this. On some places it was pushed (mostly older), on others (recent one's) it was not. Probably, the reason is, if your `.remarks.md` file changes then you have to regenerate this `.ecschema.md` file to see the latest changes and one may forget about this.  Our build process automatically does this stuff for us and its output is used in production. On the other hand, the benefit of pushing this file is that we can visually see the output here.

@ColinKerr, should we make a concrete policy for this? or keep it optional?

Reply:
I remoed the file after talking to Colin.

Reply:
Sounds good!